droid-hal-device.inc

```diff
@@ -315,6 +315,13 @@ hybris/mer-kernel-check/mer_verify_kernel_config \
 # Check if local overrides for tool mk files exist, otherwise dhd defaults are used
 if test -f rpm/helpers/mkbootimg.mk; then
     export MKBOOTIMG_MK=$(pwd)/rpm/helpers/mkbootimg.mk
+else
+    if [ $(grep 'PLATFORM_VERSION := ' \
```

Comment:
A comment would be handy here explaining that starting with Android 7 mkbootimg is a Python script, and no longer requires make/building (IIUC)